### PR TITLE
Fixed lint, added lint and style to test script

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

**Comment by joefitter** [comment]
@easternbloc @JoeChapman @danielepolencic @daniel-ac-martin - linting errors - we weren't running lint or style on travis

File: package.json [hunk 1]
```diff
@@ -10,8 +10,9 @@
   "homepage": "https://github.com/UKHomeOffice/hof-controllers",
   "author": "HomeOffice",
   "scripts": {
-    "test": "NODE_ENV=test mocha",
+    "test": "npm run lint && npm run style && npm run mocha",
```
Comment (easternbloc) [inline package.json]: 👍

Comment (danielepolencic) [inline package.json]: This is probably just personal preference, but I'd leave `NODE_ENV` in `test`since this is where I'd expect to see it.

**Comment by danielepolencic** [comment]
LGTM 👍

File: package.json (hunk 1 above)
Comment (joefitter) [inline package.json]: yeah I think you're right, will amend

**Comment by joefitter** [comment]
@JoeChapman @easternbloc @daniel-ac-martin mind giving this a 👍 ?

**Comment by JoeChapman** [comment]
LGTM

**Pull request merged**